### PR TITLE
Add tests for feed/feed.html

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

**Comment by mcking65** [comment]
Hmm, looks like the bocoup-feed branch was way behind bocoup? 33 files and nearly 200 commits. Could yu rebase before we review?

File: test/util/assertAriaDescribedby.js [hunk 1]
```diff
@@ -11,37 +11,52 @@ const assert = require('assert');
...
-    let el = document.querySelector('#' + id);
-    return el.innerText;
-  }, descriptionId);
+  for (let index = 0; index < elements.length; index++) {
 
-  assert.ok(
-    descriptionText,
-    'Element with id "' + descriptionId + '" should contain description text according to attribute "aria-describedby" on element: ' + elementSelector
-  );
+    let ariaDescribedbyExists = await t.context.session.executeScript(async function () {
```
Comment (jessebeach) [inline test/util/assertAriaDescribedby.js]: `const`

File: test/util/assertAriaDescribedby.js [hunk 2]
```diff
@@ -11,37 +11,52 @@ const assert = require('assert');
...
+    assert.ok(
+      descriptionValue,
+      '"aria-describedby" attribute should have a value on element: ' + elementSelector
+    );
+
+    const descriptionIds = descriptionValue.split(' ');
+
+    for (let descriptionId of descriptionIds) {
+
+      let descriptionText = await t.context.session.executeScript(async function () {
```
Comment (jessebeach) [inline test/util/assertAriaDescribedby.js]: `const`

File: test/tests/feed.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  }
+});
+
+ariaTest('', exampleFile, 'article-aria-setsize', async (t) => {
+  t.plan(30);
+
+  await navigateToFeed(t);
+  await waitForArticlesToLoad(t);
+
+  let articles = await t.context.session.findElements(By.css(ex.articleSelector));
```
Comment (jessebeach) [inline test/tests/feed.js]: `const`, `const`

**Pull request merged**